help-center/faq/cloud-faq.md

```diff
@@ -0,0 +1,73 @@
+---
+id: cloud-faq
+title: Memgraph Cloud - Frequently asked questions
+sidebar_label: Memgraph Cloud
+slug: /faq/cloud
+---
+
+## Pricing any payment
+
+
+### What is the pricing?
+
+You can estimate the cost of Memgraph Cloud's service by selecting your cloud region and instance size with our [Cost Calculator](https://cloud.memgraph.com/pricing) or you can check them out at [Project rates](/docs/memgraph-cloud/payment#project-rates))
+
+
+### How can I use a coupon that I got for Memgraph Cloud?
+First you need to add a credit card to your account. Then you can apply the coupon.
+
+### What will happen to my instance after the free trial?
+It will be stopped for next 7 days. If you wish to continue, add a payment method.
+
+## Projects
+
+### How many projects and snapshots can I create? Why I can’t create more than 3 projects? 
+That is the initial limit for new users. If want to create more projects, [let us know](/help-center).
+
+### Why I can’t create more than 5 snapshots? 
+That is the initial limit for new users. If want to create more snapshots,  [let us know](/help-center).
+
+### How do I upgrade a RAM instance? 
```

Comment:
https://memgraph.com/docs/memgraph-cloud/cloud-projects#resize-a-project

I would link to this process...